app/views/claims/_school_search.html.erb

```diff
@@ -33,7 +33,7 @@
         Enter the school name. Use at least four characters.
       </span>
 
-      <% if school_id_param == :claim_school_id %>
+      <% if school_id_param == :claim_school_id && !params.has_key?(:additional_school) %>
```

Comment:
Technically, `has_key?` is deprecated in favour of `key?`, but doesn't look like it'll be be removed any time soon according to https://github.com/rubocop-hq/rubocop/issues/3224.

Reply:
Yeah, I clocked that, but I think `has_key?` is more expressive

Reply:
Fine, if you want to argue with Matz's opinion... ;)

Reply:
Also, we currently call `key?` 9 times in the codebase and `has_key?` zero... just saying!